libcxx/test/benchmarks/vector_operations.bench.cpp

```diff
@@ -14,6 +14,8 @@
 #include <deque>
 #include <functional>
 #include <vector>
+#include <string>
+#include <memory>
```

Comment:
```suggestion
#include <deque>
#include <functional>
#include <memory>
#include <string>
#include <vector>
```

Reply:
Thanks. I've adjusted the order of headers as per your suggestion.